core/grants.go

```diff
@@ -118,13 +124,15 @@ func (a *API) listGrants(ctx context.Context) (map[string]interface{}, error) {
 				return nil, errors.Wrap(err)
 			}
 
-			grant := apiGrant{
-				GuardType: g.GuardType,
-				GuardData: data,
-				Policy:    g.Policy,
-				CreatedAt: g.CreatedAt,
+			if !g.Protected { // is this right?
```

Comment:
I like the idea of showing them so the user can see what the true security boundary surface area looks like. But that might mean more work for dashboard to show them grayed-out or whatever, somehow indicate that they're protected.

@jeffomatic @dominic what do you think? Ultimately I am okay with this either way depending on what the product folks think.